Description: adding version file to test
 Make test files require the version file defined in lib folder.
 .
 ruby-redis-rack (1.5.0-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- ruby-redis-rack-1.5.0.orig/test/redis/rack/version_test.rb
+++ ruby-redis-rack-1.5.0/test/redis/rack/version_test.rb
@@ -1,5 +1,5 @@
 require 'test_helper'
-
+require 'version'
 describe Redis::Rack::VERSION do
   it 'returns current version' do
     Redis::Rack::VERSION.must_equal '1.5.0'
